plugin.video.regiotv/resources/lib/addon.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+''' This is the actual Regio TV video plugin entry point '''
+
+from __future__ import absolute_import, division, unicode_literals
+from routing import Plugin
+from data import CHANNELS
+
+# pylint: disable=invalid-name
+plugin = Plugin()
+
+
+@plugin.route('/')
+def main_menu():
+    ''' The Regio TV plugin main menu '''
+    from xbmcgui import ListItem
+    import xbmcplugin
+
+    xbmcplugin.setContent(plugin.handle, content='files')
+    xbmcplugin.setPluginCategory(plugin.handle, category='Zenders')
+    xbmcplugin.setPluginFanart(plugin.handle, image='DefaultAddonPVRClient.png')
+    xbmcplugin.addSortMethod(plugin.handle, sortMethod=xbmcplugin.SORT_METHOD_UNSORTED)
+    xbmcplugin.addSortMethod(plugin.handle, sortMethod=xbmcplugin.SORT_METHOD_LABEL)
+
+    listing = []
+    for channel in CHANNELS:
+        is_folder = False
```

Comment:
the `is_folder` and `is_playable` values never change. So why all the checks on the values?

Reply:
They don't change at the moment, but they may in the future.
The code resembles other code, and I find it useful.